Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
That makes sense. Then the only way to include it is to include somewhat implementation code (though very basic implementation code) that parses the cookies as a computed property.

I'll make a PR into this one that contains code to do that.